packages/codec/lib/interface/decoders/contract.ts

```diff
@@ -382,10 +477,21 @@ export class ContractInstanceDecoder {
     return await this.wireDecoder.getCode(address, block);
   }
 
-  //EXAMPLE: to watch a.b.c[d][e], use watchMappingKey("a", "b", "c", d, e)
-  //(this will watch all ancestors too, or at least ones given by mapping keys)
-  //feel free to mix arrays, mappings, and structs here!
-  //see the comment on constructSlot for more detail on what forms are accepted
+  /**
+   * Watches a mapping key; adds it to the decoder's list of watched mapping keys.
+   * This affects the results of both variables() and variable().
+   * When a mapping is decoded, only the values at its watched keys will be included in its value.
+   * Note that it is possible to watch mappings that are inside structs, arrays, other mappings, etc;
+   * see below for more on how to do this.
+   * Note that watching mapping keys is only possible in full mode; if the decoder wasn't able to
+   * start up in full mode, this method will throw an exception.
```

Comment:
Might be worth a link back to the README section that talks about this

Reply:
Is there an easy way to make links like that?